qds_sdk/commands.py

```diff
@@ -87,10 +88,28 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+
+        # vars to keep track of actual logs bytes (err, tmp) and new bytes seen in each iteration
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
```

Comment:
One last thing from my side.

Can you pop "print-live-logs" from the kwargs here. We are not doing it during runaction and sending this argument to the API. To be consistent in the print-logs popping which happened in runaction, you can use positional argument to be passed here or simply pop before create like we do it in submitaction(). based on what approach you feel is the best, keep the '''print_logs = args.pop("print_logs")''' line from runaction together here. I.e like submitaction, before issuing a create, pop all arguments not required and later use them. 

I think while testing it, API must have ignored the extra argument. 

There are two review comments which looks active (including this one) and once we have closure on these, I can merge the changes over the weekend.